code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -41,30 +41,30 @@
 
 	if(shoes)
 		if(mobility_flags & MOBILITY_STAND)
-			if(loc != OldLoc)
-				if(!has_gravity(loc))
-					return
-				var/obj/item/clothing/shoes/S = shoes
+			if(!has_gravity(loc))

```

Comment:
Check that they're floating instead of having gravity while you're touching on this